src/main/java/com/feedzai/commons/sql/abstraction/dml/dialect/SqlBuilder.java

```diff
@@ -498,6 +498,17 @@ public static StringAgg stringAgg(final Expression column) {
         return StringAgg.stringAgg(column);
     }
 
+    /**
+     * The With clause.
+     *
+     * @param name The name to be assigned to the expression.
+     * @param expression The expression.
+     * @return The With clause.
+     */
+    public static With with(final String name, final Expression expression) {
```

Comment:
Should `name` be called `alias` instead?

Reply:
Yes

Reply:
Yes, but... you did not change. :)

Reply:
Yup, I forgot to change on the SQLBuilder. Done :)